### DISCUSSION TITLE
CalendarEvent additional attributes

### DISCUSSION THREAD

**Discussion by marksie1988** [home-assistant/architecture#963] [Entity Models] [answered]
# Background

I have been developing the atomic-calendar-revive card for some years now, however when the calendar event was changed to allow for less breaking change some features that were useful for the card were removed.

This has meant that functionality that the users of the card previously had is no longer possible and it limits future features for this card and potentially other cards.

# Proposal

Add the following options to the CalendarEvent to allow for more functionality outside of the built-in HA calendar

* Status: accepted / tentative / cancelled / declined state of events
* Link: the url to the event
* Organiser: information on the organiser of the event
* Attendees: information on the invited attendees

**Comment by Jc2k** [comment]
Can you give some more background on how you'll implement them perhaps?

**Reply by marksie1988** [reply]
I thought that this was to request having these added to the existing CalendarEvent, im  not familiar with the HA backend for the calendars and what would be required. However i know the data is available but the new CalendarEvent class removed it when it was normalised.

**Reply by Jc2k** [reply, marked as answer]
Oh, no it's not. Feature requests go on the forum.

This is where developers discuss changes they are going to make themselves. We'd then ask them things like are these mandatory fields, do we need to differentiate between unsupported and unset fields, which integrations benefit from the change, etc. If you have good answers it might get approved. If a change is approved you get to implement it.

**Reply by marksie1988** [reply]
not sure if it makes any difference but this was previously discussed in https://github.com/home-assistant/core/issues/71581 that was raised. possibly @allenporter may be able to assist here?

**Reply by allenporter** [reply]
> not sure if it makes any difference but this was previously discussed in https://github.com/home-assistant/core/issues/71581 that was raised. possibly @allenporter may be able to assist here? 

This doesn't change the answer above, this is not a feature request forum.
